doc/essentials/EnergyPlusEssentials.tex

```diff
@@ -0,0 +1,2260 @@
+needs to be installed separately. It is also open source and is available
+from \href{https://github.com/NREL/EP-Launch}{GitHub}, and it is
+documented on \href{https://ep-launch.readthedocs.io/en/}{readthedocs}
+or in the docs folder on GitHub. EP-Launch 3 works on Windows, MacOS,
+and Linux systems and is written in Python.
+
+While both EP-Launch 2 and EP-Launch 3 do many of the same functions,
+the interface is quite different. For now, EP-Launch 2 allows groups
+of files to be run together and has access to some utilities that
+the newer version does not. EP-Launch 3 works across multiple platforms
+and is a built from the ground up to be flexible and extensible so
+that individuals can make their own workflows that run whatever programs
+they need to run.
+
+\subsection{IDF and JSON syntax}
+
+EnergyPlus has two different input file formats that can be used to
+describe the building and system that is simulated. The file extensions
+for the two formats are IDF and epJSON. For both input files, the
+numeric inputs are in SI units (International System of Units often
+called metric units). 
+
+\subsubsection*{IDF}
+
+The legacy file format is a text-based format that describes each
+input object in series. Each input object starts with the kind of
+input object, and each value for each field follows while being separated
+by commas. The end of the input object is indicated by a semi-colon.
+Comments are indicated by an exclamation point ``!'' and anything
+after this is ignored by EnergyPlus. Commonly, an input object is
+spread over many lines in the file with one value for each field per
+line. The names of each field are not required but are usually shown
+after the value and the comma or semicolon as a special comment using
+``!-'' as an indicator. The input objects can be in any order. An
+example of an input object in an IDF file is shown below: 
+\begin{lyxcode}
+Building,
+
+~~Simple~One~Zone,~~~!-~Name
+
+~~0,~~~~~~~~~~~~~~~~~!-~North~Axis~\{deg\}
+
+~~Suburbs,~~~~~~~~~~~!-~Terrain
+
+~~0.04,~~~~~~~~~~~~~~!-~Loads~Convergence~Tolerance~Value
+
+~~0.004,~~~~~~~~~~~~~!-~Temperature~Convergence~Tolerance~Value~\{deltaC\}
+
+~~MinimalShadowing,~~!-~Solar~Distribution
+
+~~30,~~~~~~~~~~~~~~~~!-~Maximum~Number~of~Warmup~Days
+
+~~6;~~~~~~~~~~~~~~~~~!-~Minimum~Number~of~Warmup~Days
+\end{lyxcode}
+The details of this example input object are not important, but the
+use of commas, exclamation points, and the closing semi-colon are
+important. The IDF format is currently the most commonly used format
+throughout the EnergyPlus ecosystem of utilities and GUIs. The list
+of possible input objects and fields is documented in the Energy+.idd
+file.
+
+A variation on the IDF file format is the IMF file format which includes
+macros that can be used for parametric analysis or file management
+called EP-Macros. To learn more about macros see the Input Macros
+chapter of the AuxiliaryPrograms document.
```

Comment:
I like how this document is like a catch-all that simply gets users out to other more detailed sources very quickly.